build-system/amp.extern.js

```diff
@@ -35,6 +35,7 @@ window.draw3p;
 // AMP's globals
 window.AMP_TEST;
 window.AMP_TEST_IFRAME;
+window.onAsyncErrorForTesting;
```

Comment:
The usage inside `test-amp-form` complained about `function never defined` so I thought the right thing to do is this. Clearly it's not